scans/worker.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+
+from scans import decimalencoder
+import boto3
+dynamodb = boto3.resource('dynamodb')
+from scans import Response
+
+
+def worker(event, context):
+    table = dynamodb.Table(os.environ['DYNAMODB_TABLE'])
```

Comment:
Suggest replacing this with python everett config manager.  This will raise without this env var and give you namespacing.  

See: https://github.com/andrewkrug/cis/blob/profilev2/python-modules/cis_identity_vault/cis_identity_vault/__init__.py

Reply:
You'll also want to make sure table is initialized outside of the loop for performance.